openmdao/recorders/tests/test_sqlite_reader.py

```diff
@@ -912,6 +912,83 @@ def test_load_solver_cases(self):
 
         _assert_model_matches_case(case, model)
 
+
+    def test_recording_option_precedence_driver_cases(self):
+        prob = Problem()
+        model = prob.model
+        model.add_subsystem('p1', IndepVarComp('x', 50.0), promotes=['*'])
+        model.add_subsystem('p2', IndepVarComp('y', 50.0), promotes=['*'])
+        model.add_subsystem('comp', Paraboloid(), promotes=['*'])
+        model.add_subsystem('con', ExecComp('c = x - y'), promotes=['*'])
+
+        prob.driver = ScipyOptimizeDriver()
+        prob.driver.options['optimizer'] = 'SLSQP'
+        prob.driver.options['tol'] = 1e-9
+        prob.driver.options['disp'] = False
+
+        model.add_design_var('x', lower=-50.0, upper=50.0)
+        model.add_design_var('y', lower=-50.0, upper=50.0)
+        model.add_objective('f_xy')
+        model.add_constraint('c', lower=15.0)
+
+        prob.driver.add_recorder(self.recorder)
+        prob.driver.recording_options['record_desvars'] = True
+        prob.driver.recording_options['includes'] = []  # the new default according to this story
```

Comment:
looks like some extraneous comments here (4 occurrences)... 
references to "the new default" and "this story" are temporal and definitely won't make sense to a reader of the docs